### ISSUE TITLE
your label function is incorrect

### ISSUE THREAD

**Issue by yffbit** [jiewwantan/XGBoost_stock_prediction#1]
you should use the Return of next day, not the Return of current day.
`return (df['Returns'].shift(-seq_length) > 0).astype(int)`

**Comment by Robertliucd** [comment]
> you should use the Return of next day, not the Return of current day.
> `return (df['Returns'].shift(-seq_length) > 0).astype(int)`

yes,you should shift to next day,not current day to predict the returns
